extensions-core/kafka-indexing-service/src/main/java/org/apache/druid/indexing/kafka/supervisor/KafkaSupervisor.java

```diff
@@ -839,8 +839,7 @@ private void killTasksInGroup(TaskGroup taskGroup)
   {
     if (taskGroup != null) {
       for (String taskId : taskGroup.tasks.keySet()) {
-        log.info("Killing task [%s] in the task group", taskId);
-        killTask(taskId);
+        killTask(taskId, "Killing task [%s] in the task group[%s]", taskId, taskGroup.groupId);
```

Comment:
I think add a reason to explain why killing a task in a task group would be better. Maybe modify `private void killTasksInGroup(TaskGroup taskGroup)` to `private void killTasksInGroup(TaskGroup taskGroup, String reason)`.

Reply:
Good point. Fixed. Thanks!